lightning/src/util/errors.rs

```diff
@@ -46,11 +46,15 @@ pub enum APIError {
 		/// A human-readable error message
 		err: String
 	},
-	/// An attempt to call watch/update_channel returned a
-	/// [`ChannelMonitorUpdateStatus::InProgress`] indicating the persistence of a monitor update
-	/// is awaiting async resolution. Once it resolves the attempted action should complete
-	/// automatically.
-	MonitorUpdateFailed,
+	/// An attempt to call [`chain::Watch::watch_channel`]/[`chain::Watch::update_channel`]
+	/// returned a [`ChannelMonitorUpdateStatus::InProgress`] indicating the persistence of a
+	/// monitor update is awaiting async resolution. Once it resolves the attempted action should
+	/// complete automatically.
+	///
+	/// [`chain::Watch::watch_channel`]: crate::chain::Watch::watch_channel
+	/// [`chain::Watch::update_channel`]: crate::chain::Watch::update_channel
+	/// [`ChannelMonitorUpdateStatus::InProgress`]: crate::chain::ChannelMonitorUpdateStatus::InProgress
```

Comment:
`check_commits` is failing because it wants this in d782856dea865e8d6ec9bcbf19aaf79eea29a9e1.